src/Automation/SnapshotCommand.cs

```diff
@@ -27,11 +27,11 @@ static class SnapshotCommand
         /// creating the expected results format, and finding the target element to scan</param>
         /// <param name="cancellationToken">A cancellation token</param>
         /// <returns>A ScanOutput object that describes the result of the command</returns>
-        public static Task<ScanOutput> ExecuteAsync(Config config, IScanTools scanTools, CancellationToken cancellationToken)
+        public static async Task<ScanOutput> ExecuteAsync(Config config, IScanTools scanTools, CancellationToken cancellationToken)
         {
             ValidateScanParameters(config, scanTools);
 
-            return Task.Run<ScanOutput>(() => GetScanOutput(config, scanTools, cancellationToken));
+            return await Task.Run<ScanOutput>(() => GetScanOutput(config, scanTools, cancellationToken)).ConfigureAwait(false);
```

Comment:
@codeofdusk, this is what you originally had and I asked you to change it because I was convinced that it would make ExecuteAsync synchronous. I've tested this change with an injected delay in the task and ExecuteAsync is asynchronous after all. My bad for asking you to change this.